composer/datasets/cifar.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2021 MosaicML. All Rights Reserved.
```

Comment:
is there a reason we renamed `cifar10` -> `cifar`?

Reply:
I trivially added webdataset versions of cifar20 and cifar100 while converting cifar10. This is designed as base class CIFARWebDatasetHparams with three subclasses with only different class variables defining remote bucket, n_classes, etc.

Might as well have all three? I think having webdataset cifar10 is primarily useful for us as a guide for external users looking to create their own webdatasets. Alternatively we could remove cifar20 and cifar100 without pain (and rename back to cifar10.py), or remove all the webdataset cifar stuff due to their small size.

Reply:
ah ok makes sense!